### ISSUE TITLE
running tools/deploy.sh does not create metal3 namespace

### ISSUE THREAD

**Issue by yee379** [metal3-io/baremetal-operator#832]
just trying a simple install using 

    IRONIC_HOST=myhost IRONIC_HOST_IP=172.20.0.10 ./deploy.sh true true true true false

spits out a bunch of errors:

    Error from server (NotFound): error when creating "STDIN": namespaces "metal3" not found

upon inspection, this is because the 

    ${KUSTOMIZE} build "${IRONIC_SCENARIO}"

doesn't seem to include a `create namespace metal3` artifact.

**Comment by fmuyassarov** [comment]
That's right, we don't create `metal3` namespace. This is needed for Ironic deployment I believe. Probably we need to add namespace creation step before `./tools/deploy.sh <deploy-BMO> <deploy-Ironic> <deploy-TLS> <deploy-Basic-Auth> <deploy-Keepalived>` in https://github.com/metal3-io/baremetal-operator/blob/master/docs/deploying.md.

/kind documentation

**Comment by nkwangleiGIT** [comment]
I think it's renamed to namespace "capm3-system" now?
I can update the code or document to create this namespace if the name is correct.

**Comment by yeahdongcn** [comment]
With PR https://github.com/metal3-io/baremetal-operator/pull/944, this should be resovled.
